doc/release-notes/tapyrus/release-notes-0.5.0.md

```diff
@@ -0,0 +1,71 @@
+Tapyrus version 0.5.0 is now available for download at:
+  https://github.com/chaintope/tapyrus-core/releases/tag/v0.5.0
+
+Please report bugs using the issue tracker at github:
+  https://github.com/chaintope/tapyrus-core/issues
+
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+  https://github.com/chaintope/tapyrus-core/tarball/v0.5.0  # .tar.gz
+
+
+How to Upgrade
+==============
+
+If you are running a node on older version(v0.3.0, v0.4.0, v0.4.1) testnet, shut it down. Wait until it has completelyshut down. Follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-node-on-tapyrus-testnet) to sart a new Tapyrus v0.5.0 node.
+
+If you want to running a private tapyrus network, shut down all nodes. Follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-new-tapyrus-network) to sart a new Tapyrus v0.5.0 network. Please note that tapyrus-signer network should also be upgraded to v0.5.0 following [tapyrus signer network setup](https://github.com/chaintope/tapyrus-signer/blob/master/doc/setup.md#how-to-set-up-new-tapyrus-signer-network)
+
+Downgrading warning
+-------------------
+
+Tapyrus blockchain created by older versions(v0.3.0) is not compatible with v0.5.0 and vice-versa. The testnet is reset with the release of v0.5.0.
+
+Compatibility
+==============
+
+Tapyrus v0.5.0 is supported on three platforms : Linux, MacOS and Windows(WSL)
+
+0.5.0 change log
+------------------
+
+*Colored coin wallet*
```

Comment:
The main thing is support for Colored Coin scheme, with changes to the consensus rules and the addition of Colored Coin support for wallet. Please include a brief introduction to Colored Coin scheme and add a link to the documentation as details.